README.md

```diff
@@ -80,7 +80,9 @@ Google calendar is here: https://goo.gl/PkDijT
 
 ### Security Audit
 
-A third party security audit was performed by Cure53, you can see the full report [here](docs/SECURITY_AUDIT.pdf).
+There has been several third party engagements focused on Envoy security:
+* In 2018, a Security audit was performed by Cure53, you can see the full report [here](docs/security/audity_cure53_2018.pdf)
+* In 2021, fuzzing infrastructure improvements were performed by Ada Logics, you can see the full report [here](docs/security/fuzzer_improvement_adalogics_2021.pdf). 
```

Comment:
this is a little confusingly worded - perhaps something like an "...an audit of fuzzing infrastructure was performed by Ada Logics with recommendations for improvements...

Reply:
How do you like it now? I tried to make it a bit shorter as well by removing the "you can see the full report" and instead linking with "full report".

Reply:
much better thanks!